core/src/io/anuke/mindustry/editor/MapRenderer.java

```diff
@@ -15,6 +16,9 @@
 import io.anuke.mindustry.world.Block;
 import io.anuke.mindustry.world.Tile;
 import io.anuke.mindustry.world.blocks.BlockPart;
+import java.awt.Point;
```

Comment:
*Never* use any classes in `java.awt` in projects that target Android, such as Mindustry. The only main packages that should be used are `java.util` and `java.io`. `java.awt` is not available on Android, and is stripped out of the core library on desktop to reduce packaged size.

If you look around the codebase, you'll see that I use `io.anuke.arc.math.geom.Point2` when a point object is absolutely necessary. However, this is not the case (see other comments below).